tests/Behat/Context/Setup/ChannelContext.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Setup;
+
+
+use Behat\Behat\Context\Context;
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Test\Services\DefaultChannelFactoryInterface;
+
+
+final class ChannelContext implements Context
+{
+    /**
+     * @var SharedStorageInterface
+     */
```

Comment:
The convention in this plugin is to use short docblocks `/** @var SharedStorageInterface */`